.github/workflows/ci.yaml

```diff
@@ -185,9 +185,70 @@ jobs:
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
       - name: Build and push
         id: docker_build
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
+        with:
+          file: dse-68/Dockerfile.${{ matrix.jdk-version }}
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          platforms: linux/amd64
+          target: dse68
+
+  publish-dse-ubi:
+    name: Publish DSE UBI image
+    if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
```

Comment:
Do you want to make this push only if it is in our fork?

Reply:
That is the effective behavior because of the `needs` on the next line that won't run if the repo owner isn't k8ssandra. Provided I fix the next line as you pointed out.